### PR TITLE
Add setup_schemas and setup_indexes into tools/setup_riak.escript

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR continues "https://github.com/esl/MongooseIM/pull/2459"

Proposed changes include:
* extend `setup_riak.escript` to configure everything.
* don't use `./tools/setup_riak` anymore :) (less bash). Keep it for poor people, who needs it to configure riak in the real life.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[7314.1](https://travis-ci.org/esl/MongooseIM/jobs/598651894) / Erlang 22.0 / small_tests / b0814b2637bf25759493924353c40f1803a9f961
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2493/7314/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2493/7314/small_tests.22.0/small/ct_run.mongooseim@localhost.2019-10-16_14.17.27/lib.mongooseim.logs/run.2019-10-16_14.17.28/suite.log.html)

File: tools/travis-setup-db.sh [hunk 1]
```diff
@@ -173,6 +173,9 @@ elif [ "$db" = 'riak' ]; then
         $(mount_ro_volume "${SSLDIR}/mongooseim/key.pem" "/etc/riak/key.pem") \
         $(mount_ro_volume "${SSLDIR}/ca/cacert.pem" "/etc/riak/ca/cacertfile.pem") \
         $(mount_ro_volume "$TOOLS/setup_riak.escript" "/setup_riak.escript") \
+        $(mount_ro_volume "$TOOLS/mam_search_schema.xml" "/mam_search_schema.xml") \
+        $(mount_ro_volume "$TOOLS/vcard_search_schema.xml" "/vcard_search_schema.xml") \
+        $(mount_ro_volume "$TOOLS/setup_riak.escript" "/setup_riak.escript") \
```
Comment (NelsonVides) [inline tools/travis-setup-db.sh]: Why's this line twice?

Comment (arcusfelis) [reply tools/travis-setup-db.sh]: To be sure :trollface: 
good catch

**Comment by arcusfelis** [comment]
@michalwski Actually, we don't use setup_riak in docs. We have a looong list of commands there:
https://mongooseim.readthedocs.io/en/latest/advanced-configuration/database-backends-configuration/

List of bash commands is more friendly for ops, erlang script is only readable by erlang developers though...

**Pull request merged**